Templates/fitFile_temp.py

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Is this valid Python?

Reply:
Is what valid python?

Your other comments show the piece of code you are referencing above the comment but this one doesn't seem to...

Reply:
Sorry, I meant this file. There's no way to comment on an entire file in GitHub so there's a convention to just comment on the first line.

My comment expanded:

This file has a `.py` extension on it, implying you could run it through the Python interpreter and get it to do something, but the contents of it don't look like valid Python to me. Should it instead be a `.txt` file or something?